makefile

```diff
@@ -8,8 +8,14 @@ SOURCE_URL=https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
 # MAIN COMMANDS / GOALS ------------------------------------------------------------------------------------------------
 all: tmp/output/release/icd11foundation.owl
 
+clean:
```

Comment:
Is `clean` still being used?

Reply:
I just added that. I added it just as a command for convenience.

Reply:
Is it still needed now though?

Reply:
It wasn't ever strictly needed. It's just there for convenience, and I don't think it's a harm to have such utilities.. I've seen this convention in a variety of other projects, such as [in MedGen](https://github.com/monarch-initiative/medgen/blob/a5d64231c9c233cedddd9e9f01a1d21c2992e255/makefile#L29); I committed this, but this makefile goal (and much of the MedGen makefile) was copied over from a different location from work done by Chris Mungall.